package.json

```diff
@@ -40,7 +40,8 @@
   "dependencies": {
     "@abandonware/noble": "1.9.2-5",
     "fakegato-history": "^0.5.6",
-    "mqtt": "^3.0.0"
+    "mqtt": "^3.0.0",
+    "aes-ccm": "^1.0.3"
```

Comment:
IMO I think we should take this opportunity to move from supporting node >=8 and instead support node >=10. Node 8 LTS was dropped earlier this year. This means that we can use the native crypto module in node: https://nodejs.org/api/crypto.html

Reply:
will check the native crypto module